src/components/table-of-contents/_macro-options.md

```diff
@@ -20,3 +20,16 @@
 | ---- | ------ | -------- | -------------------------------------------------- |
 | url  | string | true     | The URL or path for the page or heading to link to |
 | text | string | true     | Text for the list item                             |
+
+## relatedLinks
+
+| Name      | Type               | Required | Description                                        |
+| --------- | ------------------ | -------- | -------------------------------------------------- |
+| url       | string             | true     | The URL or path for the page or heading to link to |
+| itemsList | array`<itemsList>` | false    | An array of [list item links](#itemslist)          |
+
+## printButton
```

Comment:
Question: Is this always going to be a print button?

Reply:
Good question. I had assumed this was the case but actually the ticket doesn't specify, so I've updated to be more flexible.